test/notifications_test.js

```diff
@@ -81,29 +85,40 @@ describe('notifications', function() {
 
   serverManager.inject();
 
-  beforeEach(function*() {
-    mq.connect(mqSocket);
-    yield chakram.post(subscriptionsUrl, subscription);
-  });
+  it('should mark reminder as errored if no subscription exists', function*() {
+    let hasOneMessageBeenSent = false;
+    waitForMqMessage().then(() => {
+      hasOneMessageBeenSent = true;
+    });
 
-  afterEach(function() {
-    mq.disconnect(mqSocket);
+    yield chakram.post(remindersUrl, inputs[0]);
+    yield waitUntilReminderHasStatus('family_name', 1, 'error-no-subscription');
+    expect(hasOneMessageBeenSent).deep.equal(false);
```

Comment:
no need for `deep` ; (but I'd rather use sinon and its `sinon.assert.notCalled` :) )